README.md

```diff
@@ -0,0 +1,33 @@
+# Project Name
+
+Math Magicians WebApp
+
+Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:  Make simple calculations. Read a random math-related quote.
+
```

Comment:
- [ ] **OPTIONAL**
It'd be much nicer if you included an image of the app in the README file. 😄